### DISCUSSION TITLE
Settings for Coverage on startup

### DISCUSSION THREAD

**Discussion by jrock2004** [orta/vscode-jest#1047] [Q&A]
So I want to set some settings so that every time I open up VS Code I do not have to open the test runner and click autorun and coverage. I thought I had settings but they seem to not work. When I open VSCode I do not see coverage from last test run. I have to click the 2 icons, then run the tests. What am I doing wrong?

```
{
  "jest.coverageFormatter": "DefaultFormatter",
  "jest.autoRun": {
    "watch": true
  },
  "jest.testExplorer": {
    "showInlineError": true
  },
  "jest.showCoverageOnLoad": true
}
```

Thanks

**Comment by jrock2004** [comment]
What I would think would be cool is if the test runner had a button to click that would save your settings to the config

**Comment by connectdotz** [comment]
Settings look right to me. The coverage will only show the tests executed... Did the watch mode run any tests when it started?

**Comment by cipagauta-david** [comment]
This is working for me:
`
"jest.autoRun":{"watch": false, "coverage":true}
`
